hail/python/hail/methods/statgen.py

```diff
@@ -2925,7 +2926,7 @@ def filter_alleles(mt: MatrixTable,
     If all alternate alleles of a variant are filtered out, the variant itself
     is filtered out.
 
-    **Using _f_**
+    **Using** `f`
```

Comment:
Was the original intention to italicize? If so, **Using** *`f`*  or *f*?

Reply:
f is an argument to the function, so it should always be written \`f\`. That happens to render as italic currently, but we could change the rendering of all \`\` blocks to something else, and the formatting of argument names would stay consistent.

Reply:
I see that \`f\` gives italic. I was thinking of markdown formatting, for which \*f\* would give italic.

Reply:
\`foo\` is the sphinx "default role" (https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-default_role). Roles are the things in the syntax ``:role:`foo` ``. It currently renders as italic, but it doesn't *mean* italic (that would be \*foo\*).